### PR TITLE
GH-55 Add README.md

### PR DESCRIPTION
**Pull request by CitralFlo**

### CODE REVIEW

**Comment by CitralFlo** [comment]
@coderabbitai review

File: README.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+Your messages can also contain placeholders :P
+
+## Setup
+
+To use the library, you need to add the following repository and dependency to your `build.gradle` file:
+
+```gradle
+implementation("com.eternalcode:multification-bukkit:1.1.4")
+implementation("com.eternalcode:multification-cdn:1.1.4")
```
Comment (Rollczi) [inline README.md:38]: Please better explain about how to configure CDN/okaeri

File: README.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+        };
+    }
+
+}
+```
+
+Then on enable, you can create a new instance of the `YourMultification` class and use it to send notifications:
+
+```java
+    private AudienceProvider audienceProvider = BukkitAudiences.create(this);
```
Comment (Rollczi) [inline README.md:83]: You can remove private

File: buildSrc/src/main/kotlin/Versions.kt [hunk 3]
```diff
@@ -2,7 +2,7 @@
 object Versions {
 
     const val ADVENTURE_PLATFORM_BUKKIT = "4.3.3"
-    const val ADVENTURE_API = "4.18.0-SNAPSHOT"
+    const val ADVENTURE_API = "4.17.0"
```
Comment (Rollczi) [inline buildSrc/src/main/kotlin/Versions.kt:5]: ????

File: README.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+[![Discord](https://raw.githubusercontent.com/intergrav/devins-badges/v3/assets/cozy/social/discord-plural_vector.svg)](https://discord.gg/FQ7jmGBd6c)
+
+[![Gradle](https://raw.githubusercontent.com/intergrav/devins-badges/v3/assets/cozy/built-with/gradle_vector.svg)](https://gradle.org/)
+[![Java](https://raw.githubusercontent.com/intergrav/devins-badges/v3/assets/cozy/built-with/java17_vector.svg)](https://www.java.com/)
+
+</div>
+
+# Introduction
+
+Multification is a spigot-library that allows you to easily create configurable notifications and messages inside your minecraft plugin.
```
Comment (Jakubk15) [inline README.md:17]:
```suggestion
Multification is a spigot-based library that allows you to easily create configurable notifications and messages inside your plugin.
```

File: README.md [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+The library supports sending notifications via one or multiple options:
+- customizable messages,
+- action bar notifications,
+- title and subtitle,
+- boss bar notifications,
+- and sounds,
```
Comment (Jakubk15) [inline README.md:23]:
```suggestion
- Customizable messages,
- Action bar notifications,
- Title and subtitle,
- Bossbar notifications,
- and sounds,
```

Do rozważenia, może jakieś emoji?

File: README.md [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+
+messages can be sent to:
+- multiple players,
+- one player,
+- or the console.
```
Comment (Jakubk15) [inline README.md:28]:
```suggestion
- Multiple players,
- One player,
- Console
```

File: README.md [hunk 7]
```diff
@@ -0,0 +1,101 @@
...
+After that, you can use the `multification` instance to send notifications:
+
+```java
+    multification.create()
+        .player(player.getUniqueId())
+        .notice(messages -> messages.yourMessage)
+        .send();
+```
+
+Setting up configuration is easy both in cdn and okaeri configs. To add messages to the configuration, create variable in config with class `Notice` or `BukktiNotice`. You can also use builder. After plugin deploy you can find messages in configuration file.
```
Comment (Jakubk15) [inline README.md:99]:
```suggestion
Setting up configuration is easy both in cdn and Okaeri Configs. To add messages to the configuration, create variable in config with class `Notice` or `BukkitNotice`. You can also use builder. After plugin deploy you can find messages in configuration file.
```

**Review by Jakubk15 — APPROVED** [review]

**Comment by Jakubk15** [comment]
@coderabbitai full review

**Review by VerniQ — APPROVED** [review]

**Review by igoyek — APPROVED** [review]

**Review by vLuckyyy — APPROVED** [review]

**Review by P1otrulla — APPROVED** [review]

**Pull request merged**